rollup.config.js

```diff
@@ -14,20 +14,29 @@ export default {
       file: 'dist/timepicker.umd.js',
       name: 'timepicker',
       format: 'umd',
+      globals: {
+        react: "React",
+        "prop-types": "PropTypes",
+        "styled-components": "styled"
+      },
     },
     {
       file: 'dist/timepicker.es.js',
       name: 'timepicker',
       format: 'es',
-    }
+      globals: {
+        react: "React",
+        "prop-types": "PropTypes",
+        "styled-components": "styled"
+      },
+    },
   ],
   plugins: [
     babel({
       babelrc: false,
       exclude: 'node_modules/**',
       plugins: [
-        '@babel/plugin-proposal-class-properties',
-        '@babel/plugin-external-helpers'
```

Comment:
@DominikFischer86 Could `@babel/plugin-external-helpers` not be uninstalled now that it is no longer used? Or has the usage of the same package just become implicit?